apps/passport-client/components/screens/NoWASMScreen.tsx

```diff
@@ -0,0 +1,20 @@
+import { H2, Spacer, TextCenter } from "../core";
+import { AppContainer } from "../shared/AppContainer";
+
+export function NoWASMScreen() {
+  return (
+    <>
+      <AppContainer bg="primary">
+        <Spacer h={64} />
+        <TextCenter>
+          <H2>No Web Assembly</H2>
```

Comment:
While many Zupass users may be technical enough to know about wasm, we should assume that they haven't heard of it.

As such, I'd reecomend something a little less technical and more clear such as this.
```suggestion
          <H2>Browser not supported</H2>
```